### PR TITLE
Design/lazy loader fallback

### PR DESCRIPTION
**Pull request by JosefBud**
Lazy loader fallback component. No more, no less.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ dist/
 coverage/
 .directory
 /vscode.json
+.DS_Store
```
Comment (ayan4m1) [inline .gitignore]: Do you mind putting this in a [global .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)?

Comment (JosefBud) [reply .gitignore]: Sure!

File: src/components/App/App.js [hunk 2]
```diff
@@ -26,6 +25,7 @@ import {
   UserRecipes,
   UserSettings
 } from 'pages';
+import LazyLoaderFallback from 'components/LazyLoaderFallback/LazyLoaderFallback';
```
Comment (ayan4m1) [inline src/components/App/App.js]: Does the name `SuspenseFallback` sound OK to you? It's a bit shorter and more obvious from the React API angle.

Comment (JosefBud) [reply src/components/App/App.js]: Sounds good to me, I'll make those changes.

File: src/components/LazyLoaderFallback/LazyLoaderFallback.test.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import LazyLoaderFallback from './LazyLoaderFallback';
+import { withMemoryRouter } from 'utils';
+
```
Comment (ayan4m1) [inline src/components/LazyLoaderFallback/LazyLoaderFallback.test.js]: Do

```jsx
jest.mock('components/Header/Header', () => require('utils').mockComponent('Header'));
```

here.

File: src/components/LazyLoaderFallback/__snapshots__/LazyLoaderFallback.test.js.snap [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+            className="navbar-collapse collapse"
+          >
+            <div
+              className="navbar-nav"
+              onKeyDown={[Function]}
+            >
+              <a
+                aria-current="page"
+                className="nav--link-custom px-3 nav-link active"
+                data-rb-event-key={null}
```
Comment (ayan4m1) [inline src/components/LazyLoaderFallback/__snapshots__/LazyLoaderFallback.test.js.snap]: Implementing the mocking above will make all this mess go away.

File: src/index.scss [hunk 5]
```diff
@@ -1,5 +1,5 @@
-@import "~normalize-scss/sass/normalize/import-now";
-@import "~bootstrap/scss/bootstrap";
-@import "./components/ToastDrawer/ToastDrawer";
-@import "./components/LazyLoaderFallback/LazyLoaderFallback";
-@import "./style.scss";
+@import '~normalize-scss/sass/normalize/import-now';
```
Comment (ayan4m1) [inline src/index.scss]: Looks like we should add an override to `.prettierrc` that sets doubleQuote to true for .scss files...

Comment (JosefBud) [reply src/index.scss]: This might've just been something weird on my end, some settings in VSCode or something. I'll update this.

**Pull request merged**